invenio_pidstore/alembic/f615cee99600_create_pidstore_branch.py

```diff
@@ -0,0 +1,42 @@
+#
+# This file is part of Invenio.
+# Copyright (C) 2016 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Create pidstore branch."""
```

Comment:
(BTW a cosmetic comment: now that we have changed branch labels to include `invenio_` prefix, the recipes here and elsewhere could say "create invenio pidstore branch" for consistency... but that's just cosmetics, and the extra "invenio" word seems kind of redundant, so... Just noting it down, I'd vote not to change anything, unless it really bothers somebody.)

Otherwise looks good to me, CC @lnielsen, and looking forward to seeing Travis going green once we merge and release everything :smile: